pkg/parser/parser.go

```diff
@@ -378,7 +382,7 @@ func (p *Parser) parseTerm(scope *scope) Node {
 		}
 		return lit
 	}
-	p.appendError("unexpected character " + tt.FormatDetails())
+	p.appendError("unexpected " + tt.FormatDetails())
```

Comment:
The tests for this change are in the previous commit. Either move this change there, or those tests here.

Reply:
fixed. thank you!